doc/ChangeLog

```diff
@@ -1,4 +1,265 @@
+  #2780 -- New CN matrix fails with single point sites with the new ctsm5.3 datasets.
+  #2654 -- Failing tests for izumi_nag with mpi-serial in cesm3_0_alpha02a
+  #2640 -- Update submodules to cesm3_0_beta04 versions
+  #2548 -- ne0ARCTICne30x4 grid transient failure
+  #2544 -- Failing ne0CONUSne30x8_ne0CONUSne30x8_mt12 in CESM testing
+  #2486 -- Temporarily add back a T42 dataset for CAM
+  #2379 -- Deprecate clm5_1 physics with ctsm5.2.0
+  #2294 -- use of mct_mod is deprecated
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+  Removal of the clm5_1 physics option to CLM_PHYSICS_VERSION XML variable
+
+Changes made to namelist defaults (e.g., changed parameter values):
+  Add support for new grid: ne0np4.POLARCAP.ne30x4
+  Add support for f19 and f45 16-pft surface datasets
+  Fix some finidat issues for: f19, VR grids
+  Remove more mention of vichydro in namelist defaults
+
+Notes of particular relevance for developers:
+---------------------------------------------
+
+Caveats for developers (e.g., code that is duplicated that requires double maintenance):
+   I think we should expand the shell_commands unit testing I added in FatesSetupParam.
+   usermod and testmod shell_commands can have tricky logic that's hard to get right and then
+   easy to break. Having unit testing that tests the tricky user mods (such as NEON and PLUMBER)
+   as well as this tricky FATES capability would help bring robustness, correctness and stability to this
+   part of the code. As well as the ability to make changes to this part and refactor to improve it.
+   As part of this I think adopting a bash unit tester (such as BATS) would be a good thing to do.
+   The other part is that I think we could add automated testing for this on push to PR's to master.
+   Future changes:
+    - Move FatesSetupParam to user-mod directory and activate for all FATES
+    - Add automated github test for the unit test there
+    - Adopt BATS for a bash unit test framework
+    - Add testing for NEON and PLUMBER2 shell_commands user-mod directories
+
+Changes to tests or testing:
+   Changes to lilac/sspmatrixcn tests needed for cime update
+   Clm51 tests changed to Clm60
+   C96 tests changed to C96_C96_mt232
+   Exact restart monthly length tests changed to days because of cime testing update
+   Exact restart 15 day length tests changed to 20 because of a cime testing bug
+   Some additional VR grid tests
+   FatesColdLandUse test requires a long name transient test now
+   New testmod include directory (FatesSetupParam) to setup modifying FATES parameter file, testmods requiring it use it
+   Add a unit-tester to the FatesSetupParam directory to make sure it's functions work correctly
+   PFUNIT testing build infrastructure updated to use full ESMF library
+
+Testing summary: regular + fates + ctsm_sci
```

Comment:
Just a reminder that this section is pending.

Reply:
Updated